study.md

```diff
@@ -158,5 +165,5 @@ What is a response? what does it contain? What are some common status codes in a
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
+You could use file_get_content after or you could download the content using curl -o.
```

Comment:
Sorry question might have been a little misleading. I meant just displaying the content of reddit in your terminal. You could do `curl https://www.reddit.com`